### DISCUSSION TITLE
CLI not building extended styles from config file?

### DISCUSSION THREAD

**Discussion by sps992** [tailwindlabs/tailwindcss#9026] [Help] [answered]
Hi There,
This might be a simple error on my part. However I have set up TW V3 to build from the CLI. It is all working lovely BUT...it is ignoring the  classes I have extended via the tailwind config file? Do I need to mess around with doing it PostCSS to get this to work, as I thought it would work out the box with just tailwind CLI

*Update*
Here is my config file and file hierachy...
`module.exports = {
  content: [
    "../*.php",
    "../**/*.php",
    "../*.html",
    "../**/*.html",
    "./src/js/*.js",
    "./src/css/theme.css",
  ],
  theme: {
    container: {
      center: true,
      padding: {
        DEFAULT: "1rem",
        sm: "2.5rem",
        lg: "3rem",
      },
    extend: {
      colors: {
        'primary': {  
          '50': '#989896',  
          '100': '#8E8E8C',  
          '200': '#7A7A78',  
          '300': '#656563',  
          '400': '#51514F',  
          '500': '#3C3C3B',  
          '600': '#2A2A29',  
          '700': '#181818',  
          '800': '#060606'
      },
      'secondary': {
        '50': '#F7F7F7',  
        '100': '#EDEDED',  
        '200': '#D9D9D9',  
        '300': '#C5C4C4',  
        '400': '#B0B0B0',  
        '500': '#9C9B9B',  
        '600': '#807F7F',  
        '700': '#646363',  
        '800': '#484747',  
        '900': '#2C2B2B'},
        'tertiary': {
          '50': '#E2FFD4',  
          '100': '#D6FFC2',  
          '200': '#BEFF9E',  
          '300': '#A6FF7A',  
          '400': '#8EFF57',  
          '500': '#76FF33',  
          '600': '#52FA00',  
          '700': '#40C200',  
          '800': '#2D8A00',  
          '900': '#1B5200'
        },
    },
  },
  plugins: [],
  },
},
}`

File Hierachy:
assets
|-- dist
|   └─ css
|-- src
    └─ css
|-- package-lock.json
|-- package.json
|-- tailwind.config.js

**Comment by wongjn** [comment]
Could we take a look at your config file and/or your general project setup? We might be able to spot any mistakes.

**Comment by sps992** [comment, marked as answer]
Debugged with safelist and when moving over the colours from extending to overiding the TW core colours, I fixed the issue!
